pkg/common/utils.go

```diff
@@ -28,10 +28,17 @@ const (
 	// Volume ID Expected Format
 	// "projects/{projectName}/zones/{zoneName}/disks/{diskName}"
 	// "projects/{projectName}/regions/{regionName}/disks/{diskName}"
-	volIDToplogyKey    = 2
-	volIDToplogyValue  = 3
-	volIDDiskNameValue = 5
-	volIDTotalElements = 6
+	// OR
+	// "projects/{projectName}/zones/{zoneName}/disks/{diskName}/partitions/{partition}"
+	// "projects/{projectName}/regions/{regionName}/disks/{diskName}/partitions/{partition}"
+
+	volIDToplogyKey                 = 2
+	volIDToplogyValue               = 3
+	volIDDiskNameValue              = 5
+	volIDPartitionKey               = 6
+	volIDPartitionValue             = 7
+	volIDTotalElements              = 6
+	volIDTotalElementsWithPartition = 8
```

Comment:
Can we just require all the elements while we're still alpha?  I think this conditional number of elements is going to get messy if we ever have to add new elements